README.md

```diff
@@ -9,17 +10,31 @@
 
 ### How do I build an FMU from python code?
 
-1) Download the [fmi2slave](fmi2slave.py) module.
-2) Create a new class extending the `Fmi2Slave` class declared in the `fmi2slave` module. 
-3) Run `pythonfmu-builder.jar` (Built by Github [Actions](https://github.com/NTNU-IHB/PythonFMU/actions)).
+1) Install `pythonfmu` package:
+```bash
+pip install git+https://github.com/fcollonval/PythonFMU.git
```

Comment:
Replace with NTNU-IHB

Reply:
However, your significant contribution should be credited in the README somehow

Reply:
What about a contributor section in the readme?

Reply:
Sounds good